File: codewars/cpp/factorial.cpp
// https://www.codewars.com/kata/57a049e253ba33ac5e000212/train/cpp


#include <iostream>

long long factorial(int n) {
    long long factorial = 1;

    for(int i = 1; i <= n; ++i) {
        factorial *= i;
    }

    return factorial;
}

int main() {
    int n;

    std::cin >> n;
    std::cout << factorial(n) << std::endl;

    return 0;
}

File: codewars/python/factorial.py
# https://www.codewars.com/kata/57a049e253ba33ac5e000212/train/python


def factorial(n):
    factorial = 1

    for i in range(1, n + 1):
        factorial *= i

    return factorial


if __name__ == "__main__":
    n = int(input())
    print(factorial(n))
